params/hooks_libevm.go

```diff
@@ -241,6 +237,6 @@ func (b *BlockContext) Timestamp() uint64 {
 	return b.time
 }
 
-func (b *BlockContext) GetPredicateResultsBytes() []byte {
-	return b.predicateResultsBytes
+func (b *BlockContext) Extra() []byte {
```

Comment:
I think this PR conflicts with my comment [here](https://github.com/ava-labs/coreth/pull/662#discussion_r1816857990) 

If we can extend the BlockContext here, why we avoid using full and parsed types here?